### PR TITLE
Add my impression

### PR DESCRIPTION
**Pull request by appharos**

### CODE REVIEW

**Comment by appharos** [comment]
Add my impression.

File: index.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h3>读者来稿</h3>
 
 <select class="select"></select>
 <section class="wrap">
-
+<p class="impression">感觉对Git的理解加深了很多啊.2333~~</p>
```
Comment (vanishh) [inline index.html:77]: 你这行加到第一了，6

Comment (amaruk) [reply index.html:77]: NP. Add your comments as you like.

Comment (vanishh) [reply index.html:77]: Yes, i benefit a lot from this book, thanks for your work!

**Pull request merged**